src/olympia/abuse/models.py

```diff
@@ -312,9 +312,9 @@ def process_decision(
                 if self.target_addon_id
                 else abuse_report_or_decision.addon
             ),
-            rating=abuse_report_or_decision.rating,
-            collection=abuse_report_or_decision.collection,
-            user=abuse_report_or_decision.user,
+            rating=getattr(abuse_report_or_decision, 'rating', None),
```

Comment:
Before this patch, would calling this method just raise? Or how would it have behaved before. That would be good to add to the context as well as testing to know how you'd expect it to behave outside of this patchl.

Reply:
Ah I read it wrong then, I interpreted "failed" as the approach failed not the code.